haas/cli.py

```diff
@@ -175,16 +205,16 @@ def list_projects():
     do_get(url)
 
 @cmd
-def project_add_user(project, user):
+def user_add_project(user, project):
```

Comment:
The reason I changed it (which I talked about in a previous comment), is that all of the `project_*` calls map to URLs starting with `/project/<projectname>`. If we want to be consistent with that, we can't also have the URLs start with `/auth/basic`. Having them be `user_*` means we can map them to `/auth/basic/user/<username>`.

This kind of question hints at why I keep saying we need to stop thinking of the procedural (python) API as the primary one --- it keeps us from paying attention to what the HTTP API looks like, and that's important, since we don't want to leave developers trying to work in other languages out in the cold. But I digress...